tests/iam/CMakeLists.txt

```diff
@@ -11,7 +11,9 @@ set(TARGET aosiamcpp_test)
 # Sources
 # ######################################################################################################################
 
-set(SOURCES ${CMAKE_SOURCE_DIR}/src/iam/certhandler/certhandler.cpp certhandler/certhandler_test.cpp)
+set(SOURCES ${CMAKE_SOURCE_DIR}/src/iam/certhandler/certhandler.cpp
```

Comment:
Not relevant change.

Reply:
otherwise build fails because of UT, in the following commit it is replaced with link_library to iam library.